src/multiselect/index.js

```diff
@@ -42,6 +49,32 @@ class MultiSelect extends Component {
     });
   };
 
+  handleKeyPress = (e) => {
+    const { options } = this.props;
+    const { input, selected } = this.state;
+    const isValid = (options
+      .filter(opt =>
+        opt.label
+          .toLowerCase()
```

Comment:
why lowercase ?

Reply:
Filtering it case insensitively so that the user doesn't need to think about the case while typing.